### PR TITLE
Authful

### PR DESCRIPTION
**Pull request by huebnerob**
@alfiehanssen Merging now, please review retroactively

### CODE REVIEW

File: VIMNetworking/Networking/VimeoClient/VIMSessionConfiguration.m [hunk 1]
```diff
@@ -35,7 +35,7 @@ - (instancetype)init
     if (self)
     {
         _APIVersionString = @"3.2";
-        _baseURLString = @"https://api.vimeo.com/";
+        _baseURLString = @"https://api.vimeo-staging.com/";
```
Comment (alfiehanssen) [inline VIMNetworking/Networking/VimeoClient/VIMSessionConfiguration.m]: Is this supposed to be wrapped in a macro?

File: VIMNetworking/Networking/ErrorReporting/NSError+BaseError.h [hunk 2]
```diff
@@ -36,12 +36,6 @@ extern NSString * const __nonnull BaseErrorKey;
 extern NSString * const __nonnull AFNetworkingErrorDomain; // TODO: Why is this necessary? [AH] 5/15/15
 extern NSString * const __nonnull kVimeoServerErrorDomain;
 
-typedef NS_ENUM(NSInteger, VIMErrorCode)
```
Comment (alfiehanssen) [inline VIMNetworking/Networking/ErrorReporting/NSError+BaseError.h]: Do we need the `NSError+BaseError` category anymore? Would be nice to end up with a single error category.

File: VIMNetworking/Networking/VimeoClient/VIMSessionConfiguration.m (hunk 1 above)
Comment (huebnerob) [inline VIMNetworking/Networking/VimeoClient/VIMSessionConfiguration.m]: oops :S nice catch.  This isn't used at all and should be reverted

File: VIMNetworking/Networking/ErrorReporting/NSError+BaseError.h (hunk 2 above)
Comment (huebnerob) [inline VIMNetworking/Networking/ErrorReporting/NSError+BaseError.h]: Yah, they should be merged I think

File: VIMNetworking/Networking/VimeoClient/VIMSessionConfiguration.m (hunk 1 above)
Comment (alfiehanssen) [inline VIMNetworking/Networking/VimeoClient/VIMSessionConfiguration.m]: Confirming this is not in RC 5.5?

Comment (alfiehanssen) [inline VIMNetworking/Networking/VimeoClient/VIMSessionConfiguration.m]: Also assuming you’ll revert? I’ll file a separate ticket about merging the two error categories.

Comment (huebnerob) [inline VIMNetworking/Networking/VimeoClient/VIMSessionConfiguration.m]: Reverted, and confirming no it's not in the RC

**Pull request merged**